public/app/features/plugins/sql/components/visual-query-builder/AwesomeQueryBuilder.tsx

```diff
@@ -202,36 +204,74 @@ export const raqbConfig: Config = {
 
 export type { Config };
 
+const noop = () => '';
```

Comment:
I'd like to move this into the `@grafana/ui` and then use it from there. We have many of those functions in various places. WDYT?

Reply:
This is an extremely simple function so I'm not sure if it's worthwhile

Reply:
It is extremely simple yet it has my duplicate code in many places. I just want to remove those duplicates :) Do you see any downsides to doing that?

Reply:
I don't see any downsides at all. Is it used elsewhere besides for SQL datasources? I'm unsure if this would belong somewhere like `@grafana/ui`

Reply:
I double-checked it and it is being used in 7-8 places excluding tests. And they have different types. Making something that fits all use cases for that simple thing is a bit unnecessary, I see now. So I'll skip as you suggested. ✌️